src/components/SingleReview.jsx

```diff
@@ -0,0 +1,63 @@
+import { React, useState, useEffect } from "react";
+import { useParams } from "react-router-dom";
+import { getSingleReview } from "../utils/api";
+import moment from "moment";
+
+function SingleReview() {
+  const { review_id } = useParams();
+  const [singleReview, setSingleReview] = useState({});
+
```

Comment:
let's add some loading state so we can conditionally render a message whilst the review is loading